scripts/start_ursim.sh

```diff
@@ -34,6 +34,13 @@ IP_ADDRESS="192.168.56.101"
 PORT_FORWARDING="-p 30001-30004:30001-30004 -p 29999:29999"
```

Comment:
Since dashboard server is not present on PolyscopeX, the port 29999 should not be forwarded. Maybe create another port forwarding for PolyscopeX